README.md

```diff
@@ -124,6 +124,12 @@ Turn on debug logging
 openupm --verbose ...
 ```
 
+Add [testables](https://docs.unity3d.com/Manual/cus-tests.html) when importing
+
+```
+openupm --test ...
+```
+
```

Comment:
I think these should append to the command-add section, as it only works with it.